packages/twenty-server/src/modules/person/standard-objects/person.workspace-entity.ts

```diff
@@ -23,6 +23,7 @@ import { WorkspaceIsNullable } from 'src/engine/twenty-orm/decorators/workspace-
 import { WorkspaceIsSystem } from 'src/engine/twenty-orm/decorators/workspace-is-system.decorator';
 import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
 import { MessageParticipantWorkspaceEntity } from 'src/modules/messaging/common/standard-objects/message-participant.workspace-entity';
+import { WorkspaceIndex } from 'src/engine/twenty-orm/decorators/workspace-index.decorator';
 
 @WorkspaceEntity({
   standardId: STANDARD_OBJECT_IDS.person,
```

Comment:
Do we have a use-case for indexes today? It would be nice to implement one in that PR to make sure we don't break it. 
Search will be via ts_vector so I'm not sure ts_vectors need to be indexed? (and if they do it's probably GIN type not a b-tree?)